src/modules/SerialModule.cpp

```diff
@@ -44,9 +44,10 @@
 
 */
 
-#if (defined(ARCH_ESP32) || defined(ARCH_NRF52)) && !defined(CONFIG_IDF_TARGET_ESP32S2) && !defined(CONFIG_IDF_TARGET_ESP32C3)
+#if (defined(ARCH_ESP32) || defined(ARCH_NRF52) || defined(ARCH_RP2040)) && !defined(CONFIG_IDF_TARGET_ESP32S2) &&               \
+    !defined(CONFIG_IDF_TARGET_ESP32C3)
 
-#define RX_BUFFER 128
```

Comment:
I'll have to go back and check, but didn't we run into an issue with larger serial buffers on ESP32s and had to crank it down?

Reply:
@thebentern See:

https://www.esp32.com/viewtopic.php?t=8858

Reply:
Setting to rx buffer to 256 may work now, but that suggests there was previously a problem.

Reply:
Setting the buffer to a large value means that the serial module isn't running frequently enough. Maybe the processor is being taxed and the can't get the data in time?

Reply:
> @thebentern See:
> 
> https://www.esp32.com/viewtopic.php?t=8858

Ah yes! I think it was bellyaching in the logs if I'm remembering correctly.